tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCoverageLevel.java

```diff
@@ -75,29 +75,44 @@ public void testSpecificPaths() {
         String[][] rows = {
             { "//ldml/characters/parseLenients[@scope=\"number\"][@level=\"lenient\"]/parseLenient[@sample=\",\"]", "moderate", "20" }
         };
-        Factory phf = PathHeader.getFactory(ENGLISH);
-        CoverageLevel2 coverageLevel = CoverageLevel2.getInstance(SDI, "fr");
-        CLDRLocale loc = CLDRLocale.getInstance("fr");
-        for (String[] row : rows) {
-            String path = row[0];
-            Level expectedLevel = Level.fromString(row[1]);
-            Level level = coverageLevel.getLevel(path);
-            assertEquals("Level for " + path, expectedLevel, level);
-
-            int expectedRequiredVotes = Integer.parseInt(row[2]);
-            int votes = SDI.getRequiredVotes(loc, phf.fromPath(path));
-            assertEquals("Votes for " + path, expectedRequiredVotes, votes);
-        }
+        doSpecificPathTest("fr", rows);
     }
 
     public void testSpecificPathsPersCal() {
         String[][] rows = {
             { "//ldml/dates/calendars/calendar[@type=\"persian\"]/eras/eraAbbr/era[@type=\"0\"]", "basic", "4" },
             { "//ldml/dates/calendars/calendar[@type=\"persian\"]/months/monthContext[@type=\"format\"]/monthWidth[@type=\"wide\"]/month[@type=\"1\"]", "basic", "4" }
         };
+        doSpecificPathTest("ckb_IR", rows);
+    }
+
+    public void testSpecificPathsDeFormatLength() {
+        String[][] rows = {
```

Comment:
BTW this kind of test is really nice in JUnit:

```java
    @ParameterizedTest
    @ValueSource(strings = {
      "//ldml/…",
      "//ldml/…"
    })
   public void TestSpecificXPath(xpath) { … }
```

Reply:
Interesting!